plugin/activitywatch.vim

```diff
@@ -105,6 +109,17 @@ function! s:CreateBucket()
     call HTTPPostJson(s:bucket_apiurl, l:body)
 endfunc
 
+function! s:RefreshGitBranch(localtime)
+    if a:localtime - s:last_branch_update > 5
+        let s:last_branch_update = a:localtime
+        let l:cmd_result = systemlist('git branch --show-current')[0]
```

Comment:
Like I said before, systemlist is a synchronous operation and needs to be replaced with something asynchronous such as jobstart/job_start.